.github/workflows/tests-integration.yml

```diff
@@ -33,16 +33,16 @@ jobs:
       name: Integration client creds
     secrets: inherit
 
-  run-integration-tests-cf-env-with-min-capi:
-    name: MIN CAPI
-    if: ${{ github.event_name == 'workflow_dispatch' || github.event.workflow_run.conclusion == 'success' }}
-    uses: ./.github/workflows/tests-integration-reusable.yml
-    with:
-      capi-version: min
-      run-with-client-creds: false
-      os: ubuntu-latest
-      name: Integration MIN CAPI
-    secrets: inherit
+  # run-integration-tests-cf-env-with-min-capi:
```

Comment:
I'm assuming this is commented out until it is figured out what we actually want to test for min version?  Otherwise I'd delete it

Reply:
Yes.. Once we figured out min version, this will come back.

Reply:
Motivation is a bit different, the intention is to put them back as we figure out issues.